### DISCUSSION TITLE
How can I capture the click of a child element of a composite item disabled?

### DISCUSSION THREAD

**Discussion by lucasfelixs** [ariakit/ariakit#2571] [Questions] [answered]
I have a component that uses a CompositeItem involving some elements, in a specific scenario, I need to disable the CompositeItem but keep a clickable child, despite using the accessibleWhenDisabled prop, I can't capture the click, just hover or focus event. Can I capture the click?

**Comment by diegohaz** [comment]
Hi @lucasfelixc! Could you create a CodeSandbox or StackBlitz repro?

**Reply by lucasfelixs** [reply]
Hi @diegohaz , yes, https://codesandbox.io/s/vibrant-sun-93t2s7?file=/index.js

**Reply by lucasfelixs** [reply]
I was investigating, and I noticed that the focusable prop is blocking the click, when it's true the child can't be clickable

**Reply by diegohaz** [reply]
I guess that's the correct behavior. Disabled items shouldn't be clickable. There's probably a better alternative to this. Would you mind sharing your concrete use case? Screenshots help.

**Reply by diegohaz** [reply]
> I was investigating, and I noticed that the focusable prop is blocking the click, when it's true the child can't be clickable

`focusable={false}` disables all the features provided by the underlying [Focusable](https://ariakit.org/components/focusable) component, including the way it handles the `disabled` prop. Also, you shouldn't nest button elements in the DOM.

**Reply by lucasfelixs** [reply]
Yes, it makes sense. But just to understand better, if I click on the child element, because of the bubbling principle, the first event must be for the child, right?

**Reply by lucasfelixs** [reply]
![Group 2](https://github.com/ariakit/ariakit/assets/64923932/4c199d45-da11-468f-a6db-b10c8ae9be85)
![Group 3](https://github.com/ariakit/ariakit/assets/64923932/50772ae8-7cc4-4135-aa08-97cd3a9a9b50)

This is the concrete use case, I have a card made with a composite item, and a menu that is a child, I would like to keep the menu clickable even by disabling the card

**Reply by lucasfelixs** [reply]
I imagine that the disabled prop acts in the capture phase, right?

**Reply by diegohaz** [reply]
Oh, sorry, I meant a screenshot of the UI. But, by the form you described it, how would a keyboard user access that menu?

Edit: Oh, wait, there's a screenshot of the UI there. haha!

**Reply by lucasfelixs** [reply]
It's focusable, the first tab would focus on the card, and the second tab would focus on the menu

**Reply by diegohaz** [reply, marked as answer]
I'd just make the menu a sibling of the composite item, then. Similar to the [ComboboxDisclosure](https://ariakit.org/examples/combobox-disclosure) example, where it looks like the disclosure button is inside/above the input, but it's a sibling in the DOM.

**Reply by lucasfelixs** [reply]
Yes, this is the best alternative indeed. Thank you!
